tgui/packages/tgui/interfaces/XenomorphExtractor.jsx

```diff
@@ -0,0 +1,144 @@
+
+  const {
+    organ,
+    points,
+    upgrades,
+    caste,
+    value,
+    categories,
+    current_clearance,
+    is_x_level,
+  } = data;
+  const dropdownOptions = categories;
+  const [selectedTab, setSelectedTab] = useLocalState('category', 'NONE');
+
+  return (
+    <Window width={850} height={800} theme="crtyellow">
+      <Window.Content scrollable>
+        <Section>
+          <Stack fill vertical>
+            <Stack.Item>
+              <Button
+                fluid
+                icon="eject"
+                content={
+                  !organ ? 'Eject Biomass' : 'Eject ' + caste + ' biomass'
+                }
+                disabled={!organ}
+                onClick={() => act('eject_organ')}
+              />
+            </Stack.Item>
+            <Stack.Item>
+              <Button
+                fluid
+                icon="eject"
+                content={
+                  !organ
+                    ? 'Process Biomass'
+                    : 'Process Biomass, Expected value : ' + value
+                }
+                disabled={!organ}
+                onClick={() => act('process_organ')}
+              />
+            </Stack.Item>
+            <Stack.Item>
+              <NoticeBox info>Biological Matter : {points}</NoticeBox>
+            </Stack.Item>
+          </Stack>
+        </Section>
+        <Section title="Biological Material">
+          {!organ ? (
+            <NoticeBox danger>
+              Recepticle is empty, analyzing is impossible!
+            </NoticeBox>
+          ) : (
+            <NoticeBox notice>Biomass accepted. Ready to analyze.</NoticeBox>
+          )}
+        </Section>
+        <Divider />
+        <Section title={<span> Select Technology to print.</span>}>
+          <Box ml={1}>
+            <Dropdown
+              selected={selectedTab}
+              options={dropdownOptions}
+              color={'#876500'}
+              onSelected={(value) => setSelectedTab(value)}
+            />
+          </Box>
+          <Flex height="200%" direction="row">
+            <Flex.Item>
+              <Box m={2} bold>
+                {selectedTab !== 'NONE' && (
+                  <LabeledList>
+                    {upgrades.map((upgrades) =>
+                      upgrades.category === selectedTab ? (
+                        <LabeledList.Item
+                          key={upgrades.name}
+                          label={
+                            <Box width={25}>
+                              <h3 style={{ color: '#ffbf00' }}>
+                                {upgrades.name}
+                              </h3>
+                            </Box>
+                          }
+                          buttons={
+                            <Box preserveWhitespace>
+                              {' '}
+                              <Button
+                                fluid={1}
+                                content={'Print ' + '  (' + upgrades.cost + ')'}
```

Comment:
Just a heads up that after https://github.com/cmss13-devs/cmss13/pull/6244 buttons should not use the content property. Instead their content needs to be as a child e.g.

```js
<Button content="Bioscan" />
```

Needs to instead be:
```js
<Button>Bioscan</Button>
```